### PR TITLE
CI-95 Add Simple Api

### PR DESCRIPTION
**Pull request by fridiculous**

### CODE REVIEW

File: k8s/base/kustomization.yaml [hunk 1]
```diff
@@ -1,9 +1,18 @@
 kind: Kustomization
 apiVersion: kustomize.config.k8s.io/v1beta1
+
 commonLabels:
   app: carrera-api
+
 namePrefix: carrera-
+
+configMapGenerator:
```
Comment (fridiculous) [inline k8s/base/kustomization.yaml]: nice

File: .circleci/config.yml [hunk 2]
```diff
@@ -96,6 +99,14 @@ jobs: # a collection of steps
 
       - setup_remote_docker #setup remote docker
 
+      - run:
+          name: Download and install pack
+          command: |
+            curl -o pack.tgz -L https://github.com/buildpack/pack/releases/download/v0.2.1/pack-v0.2.1-linux.tgz
```
Comment (fridiculous) [inline .circleci/config.yml]: would it be simpler to use a circle orb?  https://circleci.com/orbs/registry/orb/circleci/heroku ?

**Comment by fridiculous** [comment]
in general LGTM 
also `express-winston@3.2.1` has a vulernability

**Comment by fridiculous** [comment]
Given the feedback from the team, let's remove buildpacks for now and just keep the packaging with docker

**Comment by TimMensch** [comment]
Merge completed in a rebased branch.

**Pull request closed without merging**